upsertGitHubTag-f79c4f36-3e61-43f4-8f6c-0b2e0e9774d7/deployment/index.js

```diff
@@ -118,23 +144,8 @@ function processEvent(event, callback) {
     // Handle installation events
     var githubEventType = requestBody['X-GitHub-Event']
     if (githubEventType === "installation_repositories") {
-        console.log('Valid installation event');
-        const username = body.sender.login;
-        const installationId = body.installation.id;
-        const repositoriesAdded = body.repositories_added;
-        var repositories = [];
-        repositoriesAdded.forEach((repo) => {
-            repositories.push(repo.full_name);
-        });
-
-        const reposString = repositories.join(",");
-        var pushPostBody = {
-            "installationId": installationId,
-            "username": username,
-            "repositories": reposString
-        };
+        const [pushPostBody,reposString] = handleInstallationRepositoriesEvent(body)
         path += "workflows/github/install";
-            
         postEndpoint(path, pushPostBody, (response) => {
```

Comment:
Aren't you supposed to not call this if pushPostBody and/or reposString are null?

Reply:
Yes. I had fixed this in a local copy of the branch but accidentally reverted it. Thanks for catching the mistake. [Fixed.](https://github.com/dockstore/lambda/blob/4a33b3c2b77be84f2191be748484c87a01b27019/upsertGitHubTag-f79c4f36-3e61-43f4-8f6c-0b2e0e9774d7/deployment/index.js#L154)